common/utils/loadUserData.js

```diff
@@ -1,11 +1,10 @@
 import { broadCastChannel } from "../store/store";
 import { parseActivityPayloadFromBackend } from "./activities";
 import {
-  DEFAULT_NB_DAYS_MISSIONS_HISTORY,
-  DEFAULT_NB_FIRST_MISSIONS,
+  DEFAULT_MONTH_RANGE_HISTORY,
   parseMissionPayloadFromBackend
 } from "./mission";
-import { DAY, now } from "./time";
+import { subMonths } from "date-fns";
```

Comment:
I used date-fns because it's a prefered solution to moment (which is very heavy) and I find their function much more simple than the one from `time.js`. But I'm not sure it won't have impact because of timezone. I didn't have any issue during my tests.